src/main/java/OrderService.java

```
@@ -1,100 +0,0 @@
-import java.io.*;
-import java.util.ArrayList;
-
-public class OrderService {
-    ArrayList<Order> orders = new ArrayList<Order>();
-
-    public OrderService() {
-        try {
-            Order foo = readOrderFromTestFile(1);
-            if (foo.isValid()) {
-                orders.add(foo);
-            }
-            Order bar = readOrderFromTestFile(2);
-            if (bar.isValid()) {
-                orders.add(bar);
-            }
-            Order foobar = readOrderFromBinaryFile(3);
-            if (foobar.isValid()) {
-                orders.add(bar);
-            }
```

empty catch blocks, exceptions should be handled instead